include/mbedtls/mbedtls_config.h

```diff
@@ -2366,6 +2366,34 @@
  */
 #define MBEDTLS_HMAC_DRBG_C
 
+/**
+ * \def MBEDTLS_LMOTS_C
+ *
+ * Enable the LMOTS one-time asymmetric hash signature algorithm.
+ *
+ * Module:  library/lm_ots.c
+ * Caller:
+ *
+ * Requires: MBEDTLS_SHA256_C
+ *
+ * Uncomment to enable the LMOTS signature algorithm.
+ */
+#define MBEDTLS_LMOTS_C
+
+/**
+ * \def MBEDTLS_LMS_C
+ *
+ * Enable the LMS stateful-hash asymmetric signature algorithm.
```

Comment:
Not sure what you mean here. Tests run if their dependencies are satisfied. You can have dependencies at the level of the test suite, each test function or each test case. So the test suite should depend on `MBEDTLS_LMS_C`, and functions that involve a private key should depend on `MBEDTLS_LMS_PRIVATE`.

Reply:
Ah, my question was that since this now isn't enabled by default, it then won't be tested by default (which means the LMS testing becomes quite limited). Is there some sort of test config file that enables more options for extended testing (or some config that the CI uses) that I should enable this option in?

Reply:
`scripts/config.py full` enables every boolean option that isn't explicitly blacklisted and a lot of test jobs use this. So you don't need to do anything special when adding a new feature that defaults off. As it stands now, tests based on the default config have public-only LMS, tests based on the full config have full LMS, and tests based on ad hoc configs have no LMS, so we have full coverage in terms of LMS-related options.